atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1124,18 +1143,33 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
         return estimatedSize;
     }
 
-    private <T, R extends ImmutableMap.Builder<Cell, T>> R getWithPostFiltering(
+    private <S extends Map<Cell, T>, T> S getWithPostFiltering(
             TableReference tableRef,
             Map<Cell, Value> rawResults,
-            R resultsAccumulator,
+            Function<Iterable<Map.Entry<Cell, T>>, S> builderBuilder,
```

Comment:
In pairing, this was a little facetious :) maybe `mapFactory` or something?

Reply:
Fixed.